### PR TITLE
Rename packages to google-fluentd-catch-all-config-structured.

### PR DESCRIPTION
**Pull request by bmoyles0117**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,23 +21,27 @@ all: pkg tar
...
-	rpmbuild -v -bb --nodeps --target noarch --define "buildroot `pwd`/${RPM_PACKAGE_DIR}/files" --define "_rpmdir `pwd`/${BUILD_DIR}/el" --define "package_version ${PACKAGE_VERSION}" --define "package_build_num 1" pkg/el/google-fluentd-catch-all-config.spec
+	rpmbuild -v -bb --nodeps --target noarch --define "buildroot `pwd`/${RPM_PACKAGE_DIR}/files" --define "_rpmdir `pwd`/${BUILD_DIR}/el" --define "package_version ${PACKAGE_VERSION}" --define "package_build_num 1" "pkg/el/${PACKAGE_NAME}.spec"
 
 tar: deb-tar el-tar
 
 # tarfile for Debian systems
 deb-tar: populate-deb
 	tar --owner=root --group=root -C ${DEB_FILES_DIR} \
-	    -czf ${PACKAGE_NAME}_${PACKAGE_VERSION}.deb.tar.gz .
+			-czf ${PACKAGE_NAME}_${PACKAGE_VERSION}.deb.tar.gz .
```
Comment (bmoyles0117) [inline Makefile]: I switched everything to tabs, as the majority of indentations were tabs previously, with a few outliers.

Comment (igorpeshansky) [reply Makefile]: Please revert this. `make` requires all commands to start with a (single) tab, but within that command, the indentation should be spaces.

File: Makefile [hunk 2]
```diff
@@ -21,23 +21,27 @@ all: pkg tar
 pkg: deb rpm
 
 deb: populate-deb
+	(cd pkg/deb/; \
+		DEBMAIL="stackdriver-agents@google.com" DEBFULLNAME="Stackdriver Agents" \
```
Comment (igorpeshansky) [inline Makefile]: These variables are not needed anymore -- just give the `--controlmaint` parameter to `dch`.

File: Makefile [hunk 3]
```diff
@@ -21,23 +21,27 @@ all: pkg tar
...
+		dch --package "${PACKAGE_NAME}" \
+		-v "${PACKAGE_VERSION}" "Automated Build")
```
Comment (igorpeshansky) [inline Makefile]: Ah, now I see why we thought this was a good idea to do in our build scripts instead. Is `"Automated build"` always a good log message? Should we provide a mechanism to override it?

**Review by igorpeshansky — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (bmoyles0117) [reply Makefile]: Eek, done.

File: Makefile (hunk 3 above)
Comment (bmoyles0117) [reply Makefile]: Done. I chose `CHANGE_TEXT`, but there's probably a better name.

File: Makefile (hunk 2 above)
Comment (bmoyles0117) [reply Makefile]: Done.

File: Makefile [hunk 4]
```diff
@@ -21,11 +22,14 @@ all: pkg tar
 pkg: deb rpm
 
 deb: populate-deb
+	(cd pkg/deb/; \
+	    dch --controlmaint --package "${PACKAGE_NAME}" \
```
Comment (igorpeshansky) [inline Makefile]: Let's unindent this line to line up with `cd`...

File: Makefile [hunk 5]
```diff
@@ -1,8 +1,9 @@
 # package google-fluentd configuration files.
 
 BASE_PACKAGE_NAME=google-fluentd
-PACKAGE_NAME=${BASE_PACKAGE_NAME}-catch-all-config
+PACKAGE_NAME=${BASE_PACKAGE_NAME}-catch-all-config-structured
 PACKAGE_VERSION=0.7
+CHANGE_TEXT="Automated Build"
```
Comment (igorpeshansky) [inline Makefile]: How about `BUILD_DESCRIPTION`?

File: Makefile [hunk 6]
```diff
@@ -21,11 +22,14 @@ all: pkg tar
 pkg: deb rpm
 
 deb: populate-deb
+	(cd pkg/deb/; \
```
Comment (igorpeshansky) [inline Makefile]: Let's use `&&` instead of `;`.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]

**Review by dhrupadb — APPROVED** [review]
LGTM :ok_hand:

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Review by qingling128 — APPROVED** [review]
LGTM.

**Pull request merged**